Doc/library/base64.rst

```diff
@@ -80,6 +82,7 @@ The modern interface provides:
 
    For more information about the strict base64 check, see :func:`binascii.a2b_base64`
 
+   Raises an :exc:`AssertionError` if the length of *altchars* is not 2.
```

Comment:
Similarly here:

``May assert or raise a :exc:`ValueError` if the length ...``

Reply:
Resolved in fbb9798b78